modules/perception/obstacle/onboard/camera_process_subnode.h

```diff
@@ -119,7 +119,7 @@ class CameraProcessSubnode : public Subnode {
   bool pb_obj_ = false;  // Objects
   apollo::canbus::Chassis chassis_;
   bool pb_ln_msk_ = false;  // Lane marking mask
-  const float ln_msk_threshold_ = 0.95f;
+  float ln_msk_threshold_ = 0.95f;
```

Comment:
Why it is not const?

Reply:
It is different in old model and new model